test/utils/session/session-serde.js

```diff
@@ -12,14 +12,13 @@ function stringifySession(session) {
   return JSON.stringify(copy);
 }
 
-function parseSession(sessionString) {
-  const sessionCopy = JSON.parse(sessionString);
-  sessionCopy.buffer = new Map(sessionCopy.buffer);
-  return Object.assign(new jstp.Session(), sessionCopy);
+function deserialize(string) {
```

Comment:
I don't like having `string` as an argument name, can we maybe consider changing it to `sessionCopy` or just `session`?

Reply:
What about `serializedSession`?

Reply:
SGTM